containers/ecr-viewer/src/app/error/auth/page.tsx

```diff
@@ -1,14 +1,12 @@
 import React from "react";
 
-import Header from "@/app/components/Header";
 import { Error } from "@/app/components/Icon";
 
 /**
  * @returns The error auth page JSX component.
  */
 const ErrorAuthPage = () => (
   <div className="height-viewport width-viewport display-flex flex-column">
-    <Header />
     <main className="display-flex flex-justify-center height-full">
```

Comment:
I know you didn't introduce this, but taking away the header here I wonder if we still need this outer div or can we consolidate the classes down into just the main 🤔  (feel free to ignore this one if you don't feel like it or it take more than one minute to do)